eclipse-collections-api/src/main/java/org/eclipse/collections/api/factory/list/MultiReaderListFactory.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2019 Goldman Sachs and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.api.factory.list;
+
+import java.util.stream.Stream;
+
+import org.eclipse.collections.api.block.function.Function0;
+import org.eclipse.collections.api.list.MultiReaderList;
+
+public interface MultiReaderListFactory
+{
+    /**
+     * @since 6.0
```

Comment:
Drop this at since tag.

Reply:
Done.  This is included in the latest commit as well